ssp

```diff
@@ -1624,6 +1628,16 @@ sub check_for_os_release_32bit {
     print_critical('         \_ Send customer this premade: "MIGRATION - 32-bit CentOS/RHEL/CL EOL"');
 }
 
+sub check_for_ea3 {
+    return if $EA4;
+    my $httpd_path = '/usr/local/apache/bin/httpd';
+    return if -l $httpd_path;
```

Comment:
Why would a single link merit knowing anything for certain?  I could see this playing _into_ a possibility or decision but not being a simple reason for an absolute return.  We would want something more concrete.

Reply:
This is a safety check because if httpd is a symlink, then we are not dealing with EA3.